release_notes/ocp_3_3_release_notes.adoc

```diff
@@ -0,0 +1,344 @@
+[[release-notes-ocp-3-3-release-notes]]
+= {product-title} 3.3 Release Notes
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+Red Hat {product-title} is a Platform as a Service (PaaS) that provides
+developers and IT organizations with a cloud application platform for deploying
+new applications on secure, scalable resources with minimal configuration and
+management overhead. OpenShift Enterprise supports a wide selection of
+programming languages and frameworks, such as Java, Ruby, and PHP.
+
+Built on Red Hat Enterprise Linux and Google Kubernetes, {product-title}
+provides a secure and scalable multi-tenant operating system for today’s
+enterprise-class applications, while providing integrated application runtimes
+and libraries. {product-title} brings the OpenShift PaaS platform to customer
+data centers, enabling organizations to implement a private PaaS that meets
+security, privacy, compliance, and governance requirements.
+
+[[ocp-33-new-features-and-enhancements]]
+== New Features and Enhancements
+
+{product-title} version 3.3 is now available. Ensure that you follow the
+instructions on upgrading your {product-title} cluster properly, including steps
+specific to this release.
```

Comment:
s/instructions on upgrading/instructions _for_ upgrading

Also, can we change the "including the steps specific to this release" to something like "as there may be steps specific to this release"? Otherwise, this paragraph isn't adding much to the topic. It's just saying "Follow the steps in this topic", which TBH is very obvious.